README.md

```diff
@@ -4,43 +4,55 @@
 
 **1. Install Jekyll and dependencies.** First thing you're going to need to do is set up Jekyll and the appropriate dependencies so you can develop locally. The easiest way to install everything is to use [RubyGems](https://rubygems.org/pages/download) and follow the instructions on the [Jekyll](http://jekyllrb.com/docs/installation/) website. In addition to the basic Jekyll installation, you also need to install a few dependency gems. After installing RubyGems on your machine, you really only need to run one basic command:
 
 `gem install jekyll`
 
 
-Ubuntu:
+On Ubuntu, run:
 ```bash
 sudo apt install ruby ruby-dev make gcc
 sudo gem install jekyll bundler
 ```
 
-**2. Run a local version of the blog.** Using Git, clone the latest version of this repository to your local machine using the following command:
+**2. Run a local version of the website.** Using Git, clone the latest version of this repository to your local machine using the following command:
 
 `git clone https://github.com/triplea-game/triplea-game.github.io.git`
 
-Then go into root of the folder and run `jekyll serve`:
+Then start the jekyll server:
 
-`cd triplea-game.github.io && jekyll serve`
-
-This will get a local version of the blog running on your machine, accessible on `localhost:4000, or whatever the terminal tells you`
+```
+cd triplea-game.github.io/
+jekyll serve`
+```
 
-**3. Read up on the documentation.** To really understand how to develop the site there are a few things you're going to need to read up on to make meaningful changes:
+This will get a local version of the website running on your machine, accessible on `localhost:4000`, or whatever the terminal tells you. 
 
-- [Jekyll](http://jekyllrb.com/docs/home/) The static site generator used for the site
-- [GitHub Pages](https://help.github.com/articles/using-jekyll-with-pages/) How the site is hosted
-- [Liquid](https://help.shopify.com/themes/liquid/basics) The templating language used with Jekyll by Shopify
+**3. Make Changes**
 
+You can then make changes to the files you cloned earlier and the local server should update automatically. You can then preview any changes on your local machine with a web browser, just refresh the page `localhost:4000`.
 
 ** Tip:** Write any content in [Markdown](https://daringfireball.net/projects/markdown/) with the [YAML front matter](http://jekyllrb.com/docs/frontmatter/) like the other pages. If you can, follow the instructions below to test your changes.
 
+**3. Commit and submit pull request**
+
+After verifying changes to your locally cloned repository, you'll need a git client of some sort installed (there are many of these, [git for windows](https://gitforwindows.org/) is an example). Follow these steps to prepare a pull-request: https://github.com/PointCloudLibrary/pcl/wiki/A-step-by-step-guide-on-preparing-and-submitting-a-pull-request
```

Comment:
"After verifying changes" sounds a little bit weird to me. Is it just me because I'm not a native speaker or is there really something weird with this phrasing?

Reply:
"your locally cloned" is the weird part. 'after verifying' is an odd  tense that AFAIK is discouraged in writing but is okay.